src/bin/uninstall.rs

```diff
@@ -9,14 +9,14 @@ struct Options {
     flag_quiet: bool,
     flag_color: Option<String>,
 
-    arg_spec: String,
+    arg_spec: Option<String>,
 }
 
 pub const USAGE: &'static str = "
 Remove a Rust binary
 
 Usage:
-    cargo uninstall [options] <spec>
+    cargo uninstall [options] [<spec>]
```

Comment:
I think you should also be able to fix this via just adding another usage line below this of `cargo uninstall -h | --help` or something like that, it's useful to continue requiring the spec here via the support docopt has